policyengine_us/variables/gov/states/al/tax/income/al_itemized_deduction.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_us.model_api import *
+
+
+class al_itemized_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Alabama itemized deductions"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://law.justia.com/codes/alabama/2021/title-40/chapter-18/article-1/section-40-18-15/"
+        "https://www.revenue.alabama.gov/ultraviewer/viewer/basic_viewer/index.html?form=2023/01/22f40schabdc_blk.pdf#page=1"
+    )
+    defined_for = StateCode.AL
+
+    def formula(tax_unit, period, parameters):
```

Comment:
Exclude the medical and salt deductions with the following code:

 deductions = [
            deduction
            for deduction in p.itemized_deductions
            if deduction
            not in [
                "salt_deduction",
                "medical_expense_deduction",
            ]
        ]